### DISCUSSION TITLE
account blocked

### DISCUSSION THREAD

**Discussion by Valentin-Shyaka** [vercel/community#1116] [Help] [answered]
Fellas why was my vercel account blocked ??

**Comment by amyegan** [comment, marked as answer]
Sorry you're running into trouble @Valentin-Shyaka. Any account that is blocked can be appealed. You should reply to the initial email sent from the Vercel Support team or send a new message to [registration@vercel.com](mailto:registration@vercel.com), and include info on what you are intending to host on Vercel.

There are many risk flags that may result in an account being blocked. You can learn more about the process here: 
- [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
- [Why has my account or deployment been blocked?](https://vercel.com/guides/why-is-my-account-deployment-blocked)
